src/libYARP_serversql/src/yarpserver.cpp

```diff
@@ -238,20 +238,21 @@ yarpserversql_API int yarpserver_main(int argc, char *argv[]) {
 
     Network yarp;
 
-    NameServerContainer nc;
-    nc.setSilent(false);
-    if (!nc.open(options)) {
+    NameServerContainer *nc = new NameServerContainer;
```

Comment:
What was the issue with this?
Anyway, I think you are leaking the object because I don't see a `delete` at the end...

Reply:
Sorry about this one, it is completely wrong and it does not even address the issue, that is a leak on `out`. I'm fixing this.

Reply:
I dropped the commit, and I see now there is no such file pointer `out` in master branch. So this issue is for devel only.